.github/workflows/labeler.yml

```diff
@@ -5,6 +5,9 @@ on:
   pull_request_target:
     types: [opened]
 
+permissions:
+  contents: read
```

Comment:
`AlexanderWert/issue-labeler@v2.3` might require some write access?

Reply:
Oh. Thanks for the warning. Because it's using `secrets.GITHUB_TOKEN` right?